entrypoint.sh

```
@@ -134,13 +135,23 @@
 
 # Start the database and wait for it to be ready before starting any other service
 if [ "$DATABASE_EMBEDDED" = "true" ]; then
-    su postgres -c "postgres -D $DATABASE_DATA_DIR" &
+    postgres -D "$DATABASE_DATA_DIR" &
     until pg_isready -h localhost -p 5432 -U postgres; do
         echo -e "\e[34m[Info] Waiting for the database to be ready...\e[0m"
```

Shellcheck (0.10.0)

[warning] In POSIX sh, echo flags are undefined.

(SC3037)